### DISCUSSION TITLE
❓ [Question] How to install trtorchc?

### DISCUSSION THREAD

**Discussion by shiyongming** [pytorch/TensorRT#627] [Q&A] [answered]
## ❓ Question

How to install trtorchc?
## What you have already tried
I use Dockerfile.21.07 build the docker. I found the trtorchc can't be used.
So I run `bazel build //cpp/bin/trtorchc --cxxopt="-DNDEBUG` to build the trtorchc.
However, it doesn't work.

## Environment

> Build information about the TRTorch compiler can be found by turning on debug messages

 - PyTorch Version (e.g., 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by narendasan** [comment, marked as answer]
(Going to make this a discussion so its easier to reference later) 

Id say the best way is to build the tarball `bazel build //:libtrtorch -c opt`. The tarball should be in bazel-bin after building is done. You can then un pack it over /usr/local to install. Alternatively your method should work, the binary should be in `bazel-bin/cpp/bin/trtorchc` but I think this is much harder to use since the libraries will not necessarily be in the path.
